docs/public-upload-flow.md

```diff
@@ -12,3 +12,4 @@ geekdocFilePath: public-upload-flow.md
 
 The following diagram describes the flow of requests:
 
+{{< svg src="static/ocis/tus-public-upload.svg" >}}
```

Comment:
Is the `static` in the path here actually necessary? In docs they are dropping it - https://gohugo.io/content-management/static-files/#readout

Reply:
Checked it out, needs the `static` path